File: tcc-rest/tantum/src/main/java/com/tantum/app/tantum/models/Disciplina.java
package com.tantum.app.tantum.models;

import java.util.List;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@NoArgsConstructor
@Getter
@Setter
public class Disciplina {

	private String nome;

	private String codigo;

	private int fase;

	private int aulas;

	private boolean obrigatoria;

	private List<String> horarios;

	private List<String> requisitos;

}

File: tcc-app/xamarin/UFSCApp/UFSCApp/UFSCApp/Model/Disciplina.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace UFSCApp.Model
{
    public class Disciplina
    {

        public string Codigo { get; set; }

        public string Nome { get; set; }

        public List<string> Requisitos { get; set; }

        public int Fase { get; set; }

        public List<string> Horarios { get; set; }

        public int CargaHoraria { get; set; }

        public bool Obrigatoria { get; set; }

    }
}
